examples/demo/README.md

```diff
@@ -0,0 +1,51 @@
+# Application connectivity demo
+
+The Terraform configuration files in this directory show how an application can be started along with a NuoDB database.
+Terraform manages the dependency of the application on the database based on the presence of values that are interpolated from `nuodbaas_database` resource attributes, like the SQL endpoint and CA PEM.
+
+# Requirements
+
+- Terraform v1.5.x or greater
+- Access to NuoDB Control Plane v2.3.x or greater
+- Docker runtime environment
+
+# Usage
+
+1. Download the contents of this repository and navigate to this directory:
+   ```
+   git clone https://github.com/nuodb/terraform-provider-nuodbaas.git
+   cd terraform-provider-nuodbaas/examples/demo
+   ```
+   Alternatively, you can download and unpack the contents of this directory specifically as follows:
+   ```
+   curl -s -L https://github.com/nuodb/terraform-provider-nuodbaas/archive/refs/heads/main.tar.gz | \
+       tar -xv --strip-components=2 terraform-provider-nuodbaas-main/examples/demo
+   cd demo
```

Comment:
This will work once the PR is merged. Currently it fails because this directory does not exist on the `main` branch.